### PR TITLE
feat: Add restart and delete operations to server list and topo

### PR DESCRIPTION
**Pull request by TianWuwt**
## Summary

Add restart and delete operations to server list and topo 

## Solution Description

image
image
image
image

### CODE REVIEW

File: ui/src/components/TopoComponent/constants.ts [hunk 1]
```diff
@@ -45,6 +45,17 @@ const zoneOperate: Topo.OperateTypeLabel = [
 ];
 
 const serverOperate: Topo.OperateTypeLabel = [
+  {
+    value: 'restartServer',
+    label: '重启server',
```
Comment (raindrop93) [inline ui/src/components/TopoComponent/constants.ts:50]: 国际化是统一处理吗？还是这里需要单独处理

File: ui/src/components/TopoComponent/index.tsx [hunk 2]
```diff
@@ -329,6 +331,20 @@ export default function TopoComponent({
         ),
       });
     }
+    if (operate === 'deleteServer') {
+      showDeleteConfirm({
+        title: '你确定要删除该server吗？',
+        // TODO
+        onOk: () => {},
```
Comment (raindrop93) [inline ui/src/components/TopoComponent/index.tsx:338]: 这里是打算如何处理？

File: ui/src/pages/Cluster/Detail/Overview/ServerTable.tsx [hunk 3]
```diff
@@ -60,7 +107,7 @@ export default function ServerTable({ servers }: { servers: API.Server[] }) {
         }
       >
         <Table
-          columns={getServerColums()}
+          columns={serverColums}
```
Comment (raindrop93) [inline ui/src/pages/Cluster/Detail/Overview/ServerTable.tsx:110]: 拼写错误

Comment (TianWuwt) [reply ui/src/pages/Cluster/Detail/Overview/ServerTable.tsx:110]: ServerTable 组件，原来是函数包含数组的，优化成直接数组展示

File: ui/src/components/TopoComponent/index.tsx (hunk 2 above)
Comment (TianWuwt) [reply ui/src/components/TopoComponent/index.tsx:338]: 等后端的接口出了后，再加下

File: ui/src/components/TopoComponent/constants.ts (hunk 1 above)
Comment (TianWuwt) [reply ui/src/components/TopoComponent/constants.ts:50]: 等整体需求周期尾部，统一处理

File: ui/src/components/TopoComponent/index.tsx (hunk 2 above)
Comment (powerfooI) [reply ui/src/components/TopoComponent/index.tsx:338]: The backend API is not ready, so it's OK to leave it empty here.

File: ui/src/pages/Cluster/Detail/Overview/ServerTable.tsx [hunk 4]
```diff
@@ -1,10 +1,22 @@
...
 import type { ColumnsType } from 'antd/es/table';
+import { flatten } from 'lodash';
+
+export default function ServerTable({
+  clusterDetail,
+}: {
+  clusterDetail: API.ClusterDetail[];
+}) {
+  const servers = clusterDetail?.servers;
+  // 是否为单节点集群： zone的个数 ===1 && OBServer的个数 == 1
```
Comment (powerfooI) [inline ui/src/pages/Cluster/Detail/Overview/ServerTable.tsx:14]: The server that is the only one server in a zone can not be deleted.

Comment (TianWuwt) [reply ui/src/pages/Cluster/Detail/Overview/ServerTable.tsx:14]: done

**Review by powerfooI — APPROVED** [review]
LGTM

**Pull request merged**